poetro_app/lib/features/poetry/data/data_sources/remote/i_poetry_remote_data_source.dart

```diff
@@ -9,7 +9,7 @@ abstract interface class IPoetryRemoteDataSource {
 
   Future<List<PoetryDTO>> getPoetriyListByCount(int count);
 
-  Future<List<PoetryDTO>> getPoetryListByKeyword(String keyword);
+  Future<List<PoetryDTO>> getPoetryListByKeyword(String keyword, int count);
```

Comment:
domain layer should not return dto, dto is data transfer object which is for data layer and  other layers should not depend in it